src/components/tree/TreeMenu.tsx

```diff
@@ -0,0 +1,27 @@
+import { FC } from "react";
+import { Props } from "../../types/treeMenuTypes";
```

Comment:
Let's define these props in the same file as the component.

Reply:
:+1: